components/multimedia/gst-plugins-good/manifests/sample-manifest.p5m

```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2016 <contributor>
+# Copyright 2018 <contributor>
```

Comment:
We usually don't include this unless some non-trivial part in the generated file changed.